### PR TITLE
README: Mention Yoe distro in supported distro list

### PR DESCRIPTION
**Pull request by kraj**
Remove angstrom as it is no longer uptodate

Fixes Issue #563

Signed-off-by: Khem Raj <raj.khem@gmail.com>

### CODE REVIEW

File: recipes-kernel/linux/files/android-drivers.cfg [hunk 1]
```diff
@@ -0,0 +1,8 @@
+CONFIG_ANDROID=y
```
Comment (agherzan) [inline recipes-kernel/linux/files/android-drivers.cfg:1]: Was this commit supposed to be part of this PR?

Comment (kraj) [reply recipes-kernel/linux/files/android-drivers.cfg:1]: in theory I wanted to open a new PR for it.

**Comment by kraj** [comment]
closing this one for opening new separate ones.

**Comment by agherzan** [comment]
Uh! I didn't mean that. Was just flagging it.

**Pull request closed without merging**